src/common/code_utils.hpp

```diff
@@ -33,6 +33,8 @@
 #ifndef OTBR_COMMON_CODE_UTILS_HPP_
 #define OTBR_COMMON_CODE_UTILS_HPP_
 
+#include "common/logging.hpp"
```

Comment:
Does this header depends on logging?

Reply:
The `SuccessOrDie` and `VerifyOrDie` refer to `otbrLog`.